### PR TITLE
Image tag

### PR DESCRIPTION
**Pull request by mbarnach**
Add the common tag for image insertion.

### CODE REVIEW

File: Tests/VauxTests/VauxTests.swift [hunk 1]
```diff
@@ -293,6 +293,33 @@ final class VauxTests: XCTestCase {
     }
   }
   
+  func testImage() {
+    var url = "my_image.png"
+    func pageWithImage() -> HTML {
+        html {
+            body {
+                image(url: url)
```
Comment (dokun1) [inline Tests/VauxTests/VauxTests.swift]: Can we correct the tabs to 2 spaces per tab here please?

Btw, this isn't an arbitrary decision - most of these code snippets are very pyramid-y, and having 2 spaces per tab width will make this code much easier to read.

File: Sources/Vaux/Builders.swift [hunk 2]
```diff
@@ -108,6 +108,13 @@ public func linkStylesheet(url: String) -> HTML {
   return HTMLNode(tag: "link", child: nil).attr("href", url).attr("rel", "stylesheet")
 }
 
+
+/// Inserts a `<img src="url"/>` element into the HTML document.
+/// - Parameter url: The url of the image to show on the webpage.
+public func image(url: String) -> HTML {
+    return HTMLNode(tag: "img", child: nil).attr("src", url)
```
Comment (dokun1) [inline Sources/Vaux/Builders.swift]: Same thing for indentation here, but this LGTM 👍

**Review by dokun1 — CHANGES_REQUESTED** [review]

**Comment by dokun1** [comment]
Thank you for all of your suggestions, @mbarnach 😄 I'm excited to see people wanting to make this better.

**Comment by mbarnach** [comment]
Indentation fixed.

**Review by dokun1 — APPROVED** [review]

**Comment by dokun1** [comment]
@mbarnach this looks good to me, but can you please merge the master branch into your branch so that I can easily merge?

**Pull request merged**